src/js/components/tasks/cards/card.tsx

```diff
@@ -159,6 +192,8 @@ const OrgCard = ({
                     orgId={org.id}
                     ownedByCurrentUser={ownedByCurrentUser}
                     isDeleting={isDeleting}
+                    reviewOrgOutOfDate={reviewOrgOutOfDate}
```

Comment:
This should also get `isRefreshingOrg={isRefreshingOrg}`, handled the same way it handles `isDeleting`.